static/js/project.js

```diff
@@ -1169,49 +1169,97 @@ function reasonDialog (item, onSuccess, titleOnly, withNotesField) {
 };
 
 function embedDialog (project) {
-    var codeArea = document.createElement('textarea'),
-        form = document.createElement('form');
+    var form = document.createElement('form'),
+        fields = document.createElement('fieldset'),
+        copyURL, copyIFrame;
 
-    form.classList.add('embed-options');
+    form.setAttribute('class', 'embed-options pure-form pure-form-aligned');
     form.innerHTML =
             '<span class="info">' + localizer.localize(
                 'Please select the elements you wish to include in the '+
                 'embedded project viewer:') + '</span>';
 
+    form.appendChild(fields);
     new Map([
         [ 'title', localizer.localize('Project title') ],
         [ 'author', localizer.localize('Project author') ],
         [ 'edit-button', localizer.localize('Edit button') ],
         [ 'pause-button', localizer.localize('Pause button') ]
     ]).forEach(function (value, key) {
-        form.innerHTML += '<span class="option"><input type="checkbox" name="' +
-            key + '" value="' + key + '" checked><label for="' + key +'">' +
-            value + '</label></span>';
+        fields.innerHTML += `<div class="pure-control-group">
```

Comment:
I tried to lean a little more on pure to do some alignment. Their default styling has a little more padding than we did.